build.fsx

```diff
@@ -29,48 +29,46 @@ let installDotNet _ = dotnetCliPath <- DotNetCli.InstallDotNetSDK dotnetCliVersi
 let gitVersionPath = !!"packages/**/GitVersion.exe" |> Seq.head
 
 let version =
-  let gitVersion = Fake.GitVersionHelper.GitVersion (fun ps -> { ps with ToolPath = gitVersionPath })
-  if Fake.EnvironmentHelper.getEnvironmentVarAsBool "APPVEYOR"
-  then
-    let version = { gitVersion with BuildMetaData = Fake.AppVeyor.AppVeyorEnvironment.BuildNumber }
-    Fake.AppVeyor.UpdateBuildVersion version.InformationalVersion
-    version
-  elif Fake.EnvironmentHelper.getEnvironmentVarAsBool "TRAVIS"
-  then
-    let version = { gitVersion with BuildMetaData = Fake.EnvironmentHelper.environVar "TRAVIS_JOB_NUMBER" }
-    version
-  else
-    { gitVersion with BuildMetaData = "local" }
+    let gitVersion = Fake.GitVersionHelper.GitVersion (fun ps -> { ps with ToolPath = gitVersionPath })
+    if Fake.EnvironmentHelper.getEnvironmentVarAsBool "APPVEYOR" then
+        let version = { gitVersion with BuildMetaData = Fake.AppVeyor.AppVeyorEnvironment.BuildNumber }
+        Fake.AppVeyor.UpdateBuildVersion version.InformationalVersion
+        version
+    elif Fake.EnvironmentHelper.getEnvironmentVarAsBool "TRAVIS" then
+        let version = { gitVersion with BuildMetaData = Fake.EnvironmentHelper.environVar "TRAVIS_JOB_NUMBER" }
+        version
+    else
+        { gitVersion with BuildMetaData = "local" }
```

Comment:
We can discuss whitespace all day long, and I don't have a strong enough preference to fight for it, but I do think that whitespace changes to big chunks of code like this should be isolated in its own PR, to make review of non-whitespace changes easier.

Reply:
As an aside, did you make these changes with some formatting tool? If so, which one, and with what settings?

If there's a standard tool to use, and it's possible to save its settings in a file and check it in, we should do that. Then, every developer can just use the formatting tool before submitting a PR, and the code will automatically be styled the same across the repo.

Reply:
It's common practice in F# to do 4 blanks. I did not plan to merge it here, just needed another diff to kick the CI